### DISCUSSION TITLE
(Unsure) Photo manager

### DISCUSSION THREAD

**Discussion by anonimo82** [doublecmd/doublecmd#1825] [Suggestions]
Hello everyone.
As in title, please, add the feature to manage pictures, with thumbnails in tabs in place of file icons and with simple image preview.
I'm not sure this would be in scope with DoubleCMD, so please excuse me if it's not and just don't consider this, but also tell me.

Thanks in advance for any help.
Regards, have a nice day.

**Comment by JonnyTech** [comment]
Menu >> Show >> Thumbnails

https://doublecmd.github.io/doc/en/help.html#iface_files

**Comment by anonimo82** [comment]
My apologies. Thanks a lot.
In my defence I have to tell, most of these features are hidden in the Italian translation.
Please, update it.

**Reply by alexx2000** [reply]
Translations are updated by volunteers.

**Reply by anonimo82** [reply]
I see, if it's no coding involved, I could update it myself.
Let me know.

**Reply by alexx2000** [reply]
https://github.com/doublecmd/doublecmd/wiki/Translation
